.github/workflows/macos-release.yml

```diff
@@ -47,8 +47,13 @@ jobs:
       # - libpng
       # - libvorbis
       # - python
+      - name: Setup Python
+        uses: actions/setup-python@0b93645e9fea7318ecaed2b359559ac225c90a2b # v5.3.0
+        with:
```

Comment:
This might be what I need on master.

Reply:
Probably. Along with some or all of the other changes I made in this PR.